app/_src/guides/customizing-transparent-proxy-configuration-in-kubernetes-mode.md

```diff
@@ -0,0 +1,434 @@
+---
+title: Customizing Transparent Proxy Configuration in Kubernetes Mode
+---
+
+{% assign docs = "/docs/" | append: page.version %}
+{% assign Kuma = site.mesh_product_name %}
+{% assign kuma-system = site.mesh_namespace %}
+{% assign controlPlane = site.set_flag_values_prefix | append: "controlPlane" %}
+{% assign transparentProxy = site.set_flag_values_prefix | append: "transparentProxy" %}
+{% assign kuma-control-plane = site.mesh_cp_name %}
+{% assign kuma-control-plane-config = kuma-control-plane | append: "-config" %}
+{% assign tproxy = site.data.tproxy %}
+
+{% assign configuration-in-configmap = "Configuration in ConfigMap <sup>(experimental)</sup>" %}
+
+The default transparent proxy configuration works well for most scenarios, but there are cases where adjustments are needed. This guide explains the various methods available for modifying the configuration, along with their limitations and recommendations on when to use each one. While these methods can be used individually, they can also be combined when necessary.
+
+{% warning %}
+It’s best to stick to one method whenever possible. Using more than one can make things more complicated and harder to troubleshoot, as it may not be clear where each setting comes from. If you need to combine methods, check the [**Order of Precedence**](#order-of-precedence) to see what the final configuration will look like based on the priority of each setting.
+{% endwarning %}
+
+## Terminology Overview
+
+- **Workload**: In this guide, **workload** refers to an application running in a Kubernetes cluster, usually represented by a [**Pod**](https://kubernetes.io/docs/concepts/workloads/pods/). Kubernetes resources like [**Deployment**](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/), [**ReplicaSet**](https://kubernetes.io/docs/concepts/workloads/controllers/replicaset/), and [**StatefulSet**](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/) are used to define and manage these workloads, resulting in one or more **Pods** where the application actually runs.
+
+- **ConfigMap**: In this guide, **ConfigMap** refers to the Kubernetes [**ConfigMap**](https://kubernetes.io/docs/concepts/configuration/configmap/) resource. It’s used to store configuration data as key-value pairs that can be easily accessed by other resources in the cluster, such as Pods, Deployments, and StatefulSets.
+
+## Method {% inc method_number %}: Control Plane Runtime Configuration
+{:#{{ tproxy.ids.guides.customize-config.k8s.methods.runtime }}}
+
+The Control Plane Runtime Configuration is a section of {{ Kuma }}'s configuration that contains important settings for how the control plane operates, including options for the transparent proxy and other key components.
```

Comment:
Ok, so this {{ Kuma }} is actually {{ site.mesh_product_name }}.

Reply:
indeed, it's easier to write docs without having to remember `site.mesh_product_name` and can just do `{{ Kuma }}`. It's also easier to read the sentences in this way